bin/nodenv-each

```diff
@@ -2,15 +2,17 @@
 #
 # Summary: Execute a command for each Node version
 #
-# Usage: nodenv each [-g] [-v] [--] <command> [arg1 arg2...]
+# Usage: nodenv each [-g] [-p] [-j] [-v] [--] <command> [arg1 arg2...]
 #
 # Executes a command for each Node version by setting NODENV_VERSION.
 # Failures are collected and reported at the end.
 #
 # Options:
-#   --help, -h      Show help/usage
-#   --system, -g    Include the system `node`.
-#   --verbose, -v   Verbose mode. Prints a header for each node.
+#   --help, -h         Show help/usage
+#   --system, -g       Include the system `node`.
+#   --parallel -p      Run everything in parallel.
```

Comment:
`-p` and `-j` are very tightly coupled.

`-j` implies `-p` so it's redundant when `-j` is present.

If we came up with a nice invocation that was semantically the same as `nodenv each -p`, but used `-j` instead...
Perhaps simply the presence of `-j` without an arg? Thoughts?

Reply:
I'll make it such that supplying `-j` at all is what kicks off the parallelism. `-j` on its own defaults to `nproc` but it should allow you to override.